content/en/docs/setup/production-environment/container-runtimes.md

```diff
@@ -45,6 +45,37 @@ check the documentation for that version.
 
 
 <!-- body -->
+## Install and configure prerequisites
+
+These instructions apply to Linux nodes only.
+
+### Forwarding IPv4 and letting iptables see bridged traffic
+
+Make sure that the `br_netfilter` module is loaded. This can be done by running `lsmod | grep br_netfilter`. To load it explicitly call `sudo modprobe br_netfilter`.
```

Comment:
```suggestion
Make sure that the `br_netfilter` kernel module is loaded. This can be done by running `lsmod | grep br_netfilter`. To load it explicitly call `sudo modprobe br_netfilter`.
```